Central and Eastern European Online Library Journals.js

```diff
@@ -0,0 +1,335 @@
+{
+	"translatorID": "19cef926-c5b6-42e2-a91c-6f2722f8b36d",
+	"label": "Central and Eastern European Online Library Journals",
+	"creator": "Timotheus Kim",
+	"target": "^https://www\\.ceeol\\.com/search",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-06-29 10:20:32"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2021 Timotheus Kim
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/article-detail?')) return "journalArticle";
+	else if (getSearchResults(doc, true)) return "multiple";
+	else return false;
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = doc.querySelectorAll('a.ng-binding');
+	for (let row of rows) {
+		let href = row.href;
+		let title = ZU.trimInternal(row.textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
+			if (items) ZU.processDocuments(Object.keys(items), invokeEMTranslator);
+		});
+	}
+	else {
+		invokeEMTranslator(doc, url);
+	}
+}
+
+function invokeEMTranslator(doc) {
+	var translator = Zotero.loadTranslator("web");
+	translator.setTranslator("951c027d-74ac-47d4-a107-9c3069ab7b48");
+	translator.setDocument(doc);
+	translator.setHandler("itemDone", function (t, i) {
+		postProcess(doc, i);
+	});
+	translator.translate();
+}
+
+//scraping abstractNote from HTML, that is not included in Embedded Metadata
+function postProcess(doc, item) {
+	let abstractEntry = ZU.xpathText(doc, '//p[@class="summary"]');
+	if (!item.abstractNote && abstractEntry) item.abstractNote = abstractEntry;
+	item.complete();
```

Comment:
Remove the Snapshot attachment - it's not needed for journal articles.

```js
for (let i = 0; i < item.attachments.length; i++) {
	if (item.attachments[i].title == 'Snapshot') {
		item.attachments.splice(i, 1);
		break;
	}
}
```

Reply:
Ok. I also had this in mind. Thanks!